shelley-ma/impl/cardano-ledger-shelley-ma.cabal

```diff
@@ -41,6 +44,9 @@ library
     nothunks,
     partial-order,
     shelley-spec-ledger,
-    small-steps
+    small-steps,
+    tasty-quickcheck,
+    tasty,
+    tasty-hunit
```

Comment:
Please don't add tests to the library

Reply:
I set up a separate test suite. so these are all gone.